src/main/java/org/broadinstitute/dsm/route/CreateClinicalDummyKitRoute.java

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsm.route;
+
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.handlers.util.Result;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.KitType;
+import org.broadinstitute.dsm.db.dao.kit.BSPKitDao;
+import org.broadinstitute.dsm.model.elasticsearch.ESProfile;
+import org.broadinstitute.dsm.model.elasticsearch.ElasticSearch;
+import org.broadinstitute.dsm.statics.RequestParameter;
+import org.broadinstitute.dsm.util.DBUtil;
+import org.broadinstitute.dsm.util.ElasticSearchUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+
+import java.util.List;
+import java.util.Optional;
+
+public class CreateClinicalDummyKitRoute implements Route {
+    private static final Logger logger = LoggerFactory.getLogger(CreateClinicalDummyKitRoute.class);
+    private static String CLINICAL_KIT_REALM = "CLINICAL_KIT_REALM";
+    private static String CLINICAL_KIT_PREFIX = "CLINICALKIT_";
+    private int REALM;
+    private static final String USER_ID = "MERCURY";
+
+    @Override
+    public Object handle(Request request, Response response) {
+        String kitLabel = request.params(RequestParameter.LABEL);
+        if (StringUtils.isBlank(kitLabel)) {
+            logger.warn("Got a create Clinical Kit request without a kit label!!");
+            return new Result(500, "Please include a kit label as a path parameter");
+        }
+        logger.info("Got a new Clinical Kit request with kit label " + kitLabel);
+        REALM = (int) DBUtil.getBookmark(CLINICAL_KIT_REALM);
```

Comment:
there is `BookmarkDao` class which you can use methods from there or add new, if they don't satisfy your requirements.